conjur/init/init_controller.py

```diff
@@ -0,0 +1,137 @@
+
+        return fetched_certificate
+
+    def get_account_info(self, conjurrc_data):
+        """
+        Method to fetch the account from the user
+        """
+        if conjurrc_data.account is None:
+            try:
+                self.init_logic.fetch_account_from_server(self.conjurrc_data)
+            # pylint: disable=broad-except,logging-fstring-interpolation
+            except Exception as error:
+                # pylint: disable=line-too-long,logging-fstring-interpolation
+                logging.debug(f"Unable to fetch the account from the Conjur server. Reason: {error}")
+                # If there was a problem fetching the account from the server, we will request one
+                conjurrc_data.account = input("Enter your organization account name: ")
+
+                if conjurrc_data.account is None or conjurrc_data.account == '':
+                    # pylint: disable=raise-missing-from
+                    raise RuntimeError("Error: account is required")
+
+    def write_certificate(self, fetched_certificate):
+        """
+        Method to write the certificate fetched from the Conjur endpoint on the user's machine
+        """
+        url = urlparse(self.conjurrc_data.appliance_url)
+        if self.conjurrc_data.cert_file is None and url.scheme == "https":
+            # pylint: disable=line-too-long
+            self.conjurrc_data.cert_file = os.path.join(os.path.dirname(conjur.constants.DEFAULT_CONFIG_FILE),
+                                            conjur.constants.CERTIFICATE_FILENAME)
+            is_file_written = self.init_logic.write_certificate_to_file(fetched_certificate,
+                                                                       self.conjurrc_data.cert_file,
+                                                                       self.force_overwrite)
+            if not is_file_written:
+                self.__ensure_overwrite_file(self.conjurrc_data.cert_file)
+                self.init_logic.write_certificate_to_file(fetched_certificate,
+                                                          self.conjurrc_data.cert_file,
+                                                          True)
+
+            sys.stdout.write(f"Wrote certificate to {conjur.constants.CERTIFICATE_FILENAME}\n\n")
+
+    def write_conjurrc(self):
+        """
+        Method to write the conjurrc configuration on the user's machine
+        """
+        is_file_written = self.init_logic.write_conjurrc(conjur.constants.DEFAULT_CONFIG_FILE,
+                                               self.conjurrc_data,
+                                               self.force_overwrite)
+        if not is_file_written:
+            self.__ensure_overwrite_file(conjur.constants.DEFAULT_CONFIG_FILE)
+            self.init_logic.write_conjurrc(conjur.constants.DEFAULT_CONFIG_FILE,
+                                           self.conjurrc_data,
+                                           True)
```

Comment:
Add explicit argument name:
```suggestion
                                           force_overwrite_flag=True)
```